generators/client-2/templates/src/main/webapp/app/account/_account.ng2module.ts

```diff
@@ -46,10 +47,12 @@ let ACCOUNT_STATES = [
 
 @NgModule({
     imports: [
+        BrowserModule,
         <%=angular2AppName%>SharedModule,
         UIRouterModule.forChild({ states: ACCOUNT_STATES })
     ],
     declarations: [
+        //<%=jhiPrefixCapitalized%>LoginModalComponent,
```

Comment:
this shouldn't be imported here it should be imported in the common module and exported from there so that it can be used in all other modules